.circleci/config.yml

```diff
@@ -260,6 +260,8 @@ jobs:
       - run: git config --global user.email "circleci@example.com"
       - run: git config --global user.name "CircleCI Deployment Bot"
       - run:
+          command: 'node scripts/compile-changelogs.js'
+          command: 'node scripts/consolidate-changelogs.js'
           command: 'node scripts/release-branch.js'
```

Comment:
Sorry I didn't notice this earlier - this syntax isn't valid. You'll need 3 separate `run` steps like:
```
- run: node scripts/compile-changelogs.js
- run: node scripts/consolidate-changelogs.js
- run: node scripts/release-branch.js
```